app/models/role.rb

```diff
@@ -0,0 +1,3 @@
+class Role < ApplicationRecord
+    validates :name, presence: true
+end
```

Comment:
Deberías configurar para que se agregue una linea adicional al final del archivo. te paso un ejemplo para configurar en VS Code:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files